app-config-exec/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import { Json, parseRawString, ParsingExtension } from '@lcdev/app-config';
+import { guessFileType } from '@lcdev/app-config/dist/config-source';
+import { exec } from 'child_process';
+import { promisify } from 'util';
```

Comment:
I would have guessed that you'd use `execa` - trying to keep dependencies low? Reason this would be important is the `preferLocal` / npm bin support (I can imagine someone wanting to use an NPM package - I guess `npx` is enough for that).

Reply:
Yeah original plan was to make this as vanilla as possible (currently at zero-dependencies). 

For the sake of simplicity my gut says to leave out the implicit `npx` logic, since prefixing with `npx`/`yarn` is trivial. Also easier to standardize if ported to other languages/ecosystems. With that said I don't have a super strong opinion on this and would be happy to use `execa` if that's preferred.

Reply:
Yeah, I'm okay with that. Easy to add as a feature as well.